apps/core/scenes/bookingDetail/sectorDetails/SectorInfoMulticity.js

```diff
@@ -17,7 +17,7 @@ function SectorInfoMulticity(props: Props) {
   return trips.map((trip, index) => (
     <React.Fragment key={index}>
       <SectorInfo data={trip} />
-      <Separator style={styles.separator} />
+      {index !== trips.length - 1 && <Separator style={styles.separator} />}
```

Comment:
I'm not sure if it's useful here, but if you would change it to `FlatList` you could use `ItemSeparatorComponent` here: https://facebook.github.io/react-native/docs/flatlist#itemseparatorcomponent

Reply:
Having a FlatList under a ScrollView has performance issues

Reply:
I see, so if the component is used somewhere with `ScrollView` parent, it make sense.